File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int slow=2,fast=2;
        int n=nums.length;
        if(n<3)
            return n;
        while(fast<n){
            if(nums[slow-2]!=nums[fast]){
                nums[slow++]=nums[fast++];
            }
            else
                fast++;
        }
        return slow;
    }
}

File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int n = nums.size();
        if(n <= 2) return n;
        int index = 2;
        for(int i=2;i<n;i++) {
            if(nums[index - 2] != nums[i]) {
                nums[index] = nums[i];
                index += 1;
            }
        }
        return index;
    }
};